safe-global/web-core
# Add policy pages

## What it solves

Resolves #1722 

## How this PR fixes it

- Adds a new page for Privacy policy, Cookie policy, Terms and Imprint
- Links those pages in the footer
- Displays the footer on those pages
- Hides the sidebar on those pages

## How to test it

1. Open the Safe
2. Go to Settings to see the Footer
3. Click on any of the three links to see the page
4. Compare to https://www.notion.so/gnosis-safe/Terms-policy-update-f3827a6f724340deae9554d24afe8c29
6. Create or Load a new safe and click on terms of use or privacy policy and observe seeing the new pages

## Screenshots

<img width="502" alt="Screenshot 2023-03-03 at 11 06 23" src="https://user-images.githubusercontent.com/5880855/222692334-b3f264a3-f512-41d5-bda5-85b2bb7de2cd.png">
